app/konnect/analytics/explorer.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Explorer
+---
+
+Explorer is an intuitive web-based interface that displays API usage data gathered by {{site.konnect_short_name}} Analytics from your data plane nodes. You can use this tool to promptly diagnose performance issues or capture essential usage metrics. Explorer also provides the option to save the output as custom reports.
+
+![Explorer Dashboard](/assets/images/products/konnect/analytics/konnect-explorer-dashboard.png){:.image-border}
+
+To begin using Explorer, go to the **Analytics** {% konnect_icon analytics %} section and select **Explorer**. 
+{% navtabs %}
+{% navtab Grouping and Filtering %}
+## Grouping and filtering
+
+This is an overview of the different grouping and filtering categories available: 
```

Comment:
I was wondering if we really need most of the description here since it does not provide any further clarification and just explain the ones that might not be as obvious. My suggestion:

* **None** (aggregates all data in the organization)
* **API Product** 
* **API Product Version** 
* **Route**
* **Application**
* **Status Code** (range from 100 to 599)
* **Status Code Group:** (1XX, 2XX, 3XX, 4XX, or 5XX)
* **Control Plane**
* **Control Plane Group**
* **Data Plane Node**
* **Gateway Services**
* **Consumer**